rosetta-source/src/main/resources/cdm-sample-files/functions/eligible-collateral/check-eligibility-eurex-vanilla-security-input.json

```diff
@@ -4,11 +4,7 @@
       {
         "asset": [
           {
-            "assetCountryOfOrigin": [
-              {
-                "value": "EU"
-              }
-            ],
+            "assetCountryOfOrigin": ["RS"],
```

Comment:
This one should be "FR" not "RS"

Reply:
The XS came about because these are EIB international securities and the ISIN country code is XS but since XS is not an official country code the issuer country code should be LU for Luxembourg.

Reply:
updated to FR